### PR TITLE
Add Prettier; Use embedded SVG for logo

### LINKED ISSUE #70: Canonical SVG logo should be optimised
The SVG logo currently contains whitespace and comments - these should be stripped out.

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Added Prettier
- Replaced uncompressed logo pulled from assets server with embedded SVG code

## QA

- Sanity check code
- Check logo displays correctly as per this codepen: https://codepen.io/barrymcgee/pen/exvobj

Fixes #70

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (anthonydillon) [inline .prettierignore]: And probably the dist folder?

**Comment by anthonydillon** [comment]
The logo is now to large:
screenshot_2019-02-01 canonical global navigation demo

**Comment by barrymcgee** [comment]
@anthonydillon Ready for re-review.

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**